Wireframe/style.css

```diff
@@ -75,15 +91,24 @@ Now laying out just the INSIDE of the repeated card/article design.
 Keeping things orderly and separate is the key to good, simple CSS.
 */
 article {
+  background-color:white;
+  color:rgb(54, 15, 87);
   border: var(--line);
-  padding-bottom: var(--space);
+  padding-bottom:20px;
   text-align: left;
   display: grid;
   grid-template-columns: var(--space) 1fr var(--space);
+  grid-template-rows: auto auto 1fr;
   > * {
     grid-column: 2/3;
   }
   > img {
     grid-column: span 3;
-  }
+    grid-row:1;
+  }}
+article a {
+  align-self: center;
+  margin-top: auto; 
 }
+h2{font-weight: bold;}
+#readme-title,#wireframe-title,#gitbranches-title{font-family: Arial, Helvetica, sans-serif;}
```

Comment:
I think this line is quite hard to read because of how much is on one line - can you think how to reformat it more clearly?